konnectd/assets/identifier/static/css/main.ed0ebb7d.chunk.css

```diff
@@ -0,0 +1,2 @@
:local("Roboto Black "),local("Roboto-Black"),url(../../static/media/roboto-latin-900.19b7a0ad.woff2) format("woff2"),url(../../static/media/roboto-latin-900.8c2ade50.woff) format("woff")}@font-face{font-family:Roboto;font-style:italic;font-display:swap;font-weight:900;src:local("Roboto Black italic"),local("Roboto-Blackitalic"),url(../../static/media/roboto-latin-900italic.7b770d6c.woff2) format("woff2"),url(../../static/media/roboto-latin-900italic.bc833e72.woff) format("woff")}@-webkit-keyframes onAutoFillStart{kopano{keepme:0}}@keyframes onAutoFillStart{kopano{keepme:0}}@-webkit-keyframes onAutoFillCancel{kopano{keepme:0}}@keyframes onAutoFillCancel{kopano{keepme:0}}#bg-thumb{background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxOTIwIiBoZWlnaHQ9IjEwODAiPjxmaWx0ZXIgaWQ9ImEiIGZpbHRlclVuaXRzPSJ1c2VyU3BhY2VPblVzZSIgY29sb3ItaW50ZXJwb2xhdGlvbi1maWx0ZXJzPSJzUkdCIj48ZmVHYXVzc2lhbkJsdXIgc3RkRGV2aWF0aW9uPSIyMCAyMCIvPjxmZUNvbXBvbmVudFRyYW5zZmVyPjxmZUZ1bmNBIHR5cGU9ImRpc2NyZXRlIiB0YWJsZVZhbHVlcz0iMSAxIi8+PC9mZUNvbXBvbmVudFRyYW5zZmVyPjwvZmlsdGVyPjxpbWFnZSBmaWx0ZXI9InVybCgjYSkiIGhyZWY9ImRhdGE6aW1hZ2UvanBlZztiYXNlNjQsLzlqLzRBQVFTa1pKUmdBQkFRRUJMQUVzQUFELzJ3QkRBQkFMREE0TUNoQU9EUTRTRVJBVEdDZ2FHQllXR0RFakpSMG9Pak05UERrek9EZEFTRnhPUUVSWFJUYzRVRzFSVjE5aVoyaG5QazF4ZVhCa2VGeGxaMlAvd0FBTENBQW9BRWNCQVJFQS84UUFHZ0FBQWdNQkFRQUFBQUFBQUFBQUFBQUFBd1lDQkFVQkFQL0VBQzRRQUFJQkF3SURCZ1lEQVFBQUFBQUFBQUVDQXdBRUVRVWhFaUl4RXhReVFWRmhCbEp4Z1pHaEpHS3gwZi9hQUFnQkFRQUFQd0J3QUJHOVJZQWRhNWhENTBOd0Y2R2hoeGpmTkdqYzRxZU0rZGR3QVBXbHI0c1V5OTNHQ1J2bkFKOVBjRDhtbWRCVVpSc0s5SEVYcU04UlJDYXFBMVYxbTZtc3JXT1NIcVd4MG8razNNbHpaSkxLZWM1enRpcnpNQXRMdnhLVi9qRmlvOFhpSyszemY4cGxRMUtUb0s4a3lSWTQyeG1vWDF4eFFBb1ZhTnhrSE85Wk1seHdLQ0ZKeWNZQjZVRFZ0VHQ0N0FQZVJyd1JudzU1dlQxM3BmOEFoWFZMYnZjc0VsM1BBWjVNb1NBVittL1NtT1RWYmUxV1B2VnlwRFNNaVBqeFlZZ2RLRnJ3Sjd0Z3VBUzNoTER5L3FEV3kwanJHZXpBTDQyejBvVVV0NFV6Y05FRzlFVWtmdXFXb2lZQlhobUtTS0NGd0NTYy9lczY2bnZGc0kxdTVTc2g4Q0p5a0QzeC9sVjdtOTdTMjVTZWJiTlpldTN0eGZMSEc1VGdVWTRlZ3pXWHBwa1NVeHFpOGZGd2JqTzlNa1MyMTFGR2swU2t4dUNpOFI1ZDl6K2FZNWJTQytFSW5KNU1rRWZTamh4OHdyb1lIekZVdFR1cmZUN2RycVZBejlGSG14OUtXdGN1WlZ2bEpqSk1rWVBHRzZmYXNwWkxvbGVPVml1Y2tZb2R4STF5M0NJVEZrNUpMakg3eFI5TTdPMjFOYmx1eXVncEpLbktndDZnMVlGMDBVeVNpMFlrWnl3azJ3ZmJIdldsRGQ2bkZweTZqYkR2SWZab21ROHB6amJCci8vWiIgaGVpZ2h0PSIxMDAlIiB3aWR0aD0iMTAwJSIvPjwvc3ZnPg==")}#bg{background-color:#fff}#bg>div{position:absolute;top:0;bottom:0;left:0;right:0;background-size:cover;background-repeat:no-repeat;background-position:50%;z-index:0}#loader{color:#fff;text-shadow:#000 0 0 1px}#bg-enhanced{opacity:0;-webkit-transition:opacity 1s;transition:opacity 1s}#bg-enhanced.enhanced{background-image:url(../../static/media/loginscreen-bg.cc3ef0e4.jpg);opacity:1}#bg-enhanced.enhanced:after{content:"";position:absolute;top:0;bottom:0;left:0;right:0;background-image:url(../../static/media/loginscreen-bg-overlay.6f9a3fb0.svg);background-size:cover;background-repeat:no-repeat;background-position:50%}
```

Comment:
Codacy found an issue: [Expected selector "::-webkit-scrollbar-track" to come before selector "::-webkit-scrollbar-button:end:increment" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)